src/Analyzers/MSTest.Analyzers/Helpers/WellKnownTypeNames.cs

```diff
@@ -36,6 +36,7 @@ internal static class WellKnownTypeNames
     public const string SystemDescriptionAttribute = "System.ComponentModel.DescriptionAttribute";
     public const string SystemIAsyncDisposable = "System.IAsyncDisposable";
     public const string SystemIDisposable = "System.IDisposable";
+    public const string SystemNullable = "System.Nullable";
```

Comment:
I suspect this should be like this:
```suggestion
    public const string SystemNullable = "System.Nullable`1";
```